myModule.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Or better, delete the entire directory in case unknown files are made, like
the myModule.md file, which was a byproduct of the actual files you made, I
believe.
tmpDir <- tempdir()
(on.exit(unlink(tmpDir)))


On Mon, Feb 1, 2016 at 3:52 PM, Alex Chubaty <notifications@github.com>
wrote:

> In myModule.md
> <https://github.com/PredictiveEcology/SpaDES/pull/249#discussion_r51478787>
> :
>
> > @@ -0,0 +1,106 @@
> > +---
>
> yes, please delete the file -- any files needed for testing should be 1)
> created in a tmpdir and 2) deleted after the test (on.exit(unlink(file)))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PredictiveEcology/SpaDES/pull/249/files#r51478787>.
>